.yarnrc.yml

```diff
@@ -0,0 +1,17 @@
+enableScripts: false
+
+enableTelemetry: 0
+
+logFilters:
+  - code: YN0004
+    level: discard
+
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
+    spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
+  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
+    spec: "@yarnpkg/plugin-constraints"
```

Comment:
I don't see any constraints in place. perhaps we should add them here? Or remove the plugin

Reply:
Good catch — we can remove the plugin for now since it assumes we have TypeScript set up (and that `package.json` is standardized itself).